
<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="UTF-8">
		<meta name="viewport" content="width=device-width, initial-scale=1.0">
		<title>2023 iDEC NKLMI-NK drug</title>
		<link rel="stylesheet" type="text/css" href="../css/main_cssstyle.css">
	 <link rel="stylesheet" type="text/css" href="../css/loader.css">
		<style>
			.collapsible {
				background: url("../images/protocols-top.png") no-repeat center center;
				
				color: black;
				cursor: pointer;
				padding: 35px;
				width: 100%;
				border: 4px solid #ffffff;
				border-radius: 15px;
				outline: none;
				font-size: 24px;
				text-align: center;
				font-weight: bold;

			}

			/*.active, .collapsible:hover {
            background-color: #555;
        }*/

			.content {
				padding: 0 18px;
				display: none;
				overflow: hidden;
				background: url("../images/protocols- context.png") no-repeat top center;
				text-align: left;
				font-size: 16px;
				height: 107px;
				line-height: 30px;
				padding-top:60px;
		
			}

.content p{text-align:center}
			header {
				background: url("../images/protocol.jpg");
				min-height: 50%;
				background-repeat: no-repeat;
				background-size: cover;
					background-position: center center;
			}

			.page-title {
				padding-top: 10em;
				padding-bottom: 1.5em;
			}

			/* footer的样式 */
			.contact-icon {
				display: flex;
				align-items: center;
				/*font-size: 3em;*/
				color: white;
			}

			.contact-icon a {
				color: white;
			}

			.contact-icon i {
				font-size: 3em;
			}
		</style>
	</head>
	<body>
	    	         <!-- 页面加载动画 -->
    <div id="loader">
        <div aria-busy="true" aria-label="Loading" role="progressbar" class="loader-container">
            <img  src="../images/load.gif">
        </div>
    </div>
		<a class="skip-main" href="#slide1-chapter1">Skip to main content</a>
		<header>
		<nav id="normal-nav">
				<!---- 正常情况下顶部导航栏Navigation bar (normal) -->
				<div class="logo">
					<a href="https://idec-teams.github.io/2023_NKLMI-NK_drug/">
						<img id="nav-logo" src="../images/logo.png"
							alt="This is our logo, but if you click it, you will return home">
					</a>
				</div>
				<div class="chapters">
					<div class="dropdown-menu cool-link">
						<a class="chapter-nav " href="https://idec-teams.github.io/2023_NKLMI-NK_drug/">
							<div class="chapter-content">
								<div class="chapter-title active ">
									Home
								</div>
							</div>
						</a>
					</div>
					<div class="dropdown-menu cool-link">
						<a class="chapter-nav " href="#">
							<div class="chapter-content">
								<div class="chapter-title active ">
									Team
								</div>
								<div class="chapter-icon">
									<i class="fas fa-sort-down"></i>
								</div>
							</div>
						</a>
						<div class="dropdown-content">
							<a href="https://idec-teams.github.io/2023_NKLMI-NK_drug/team/members.html" class="dropdown-page cool-link2">
								Members
							</a>
							<a href="https://idec-teams.github.io/2023_NKLMI-NK_drug/team/attributions.html" class="dropdown-page cool-link2">
								Attributions
							</a>
						</div>
					</div>
					<div class="dropdown-menu cool-link">
						<a class="chapter-nav " href="#">
							<div class="chapter-content">
								<div class="chapter-title active ">
									Project
								</div>
								<div class="chapter-icon">
									<i class="fas fa-sort-down"></i>
								</div>
							</div>
						</a>
						<div class="dropdown-content">
							<a href="https://idec-teams.github.io/2023_NKLMI-NK_drug/project/background.html" class="dropdown-page cool-link2">
								Background
							</a>
							<a href="https://idec-teams.github.io/2023_NKLMI-NK_drug/project/design.html" class="dropdown-page cool-link2">
								Design
							</a>
							<a href="https://idec-teams.github.io/2023_NKLMI-NK_drug/project/results.html" class="dropdown-page cool-link2">
								Results
							</a>
							<a href="images/report.pdf" class="dropdown-page cool-link2" target="_blank">
								Report
							</a>
							<a href="images/supplementary.pdf" class="dropdown-page cool-link2" target="_blank">
								Supplementary information
							</a>
						</div>
					</div>
					<div class="dropdown-menu cool-link">
						<a class="chapter-nav " href="#">
							<div class="chapter-content">
								<div class="chapter-title active ">
									Documentation
								</div>
								<div class="chapter-icon">
									<i class="fas fa-sort-down"></i>
								</div>
							</div>
						</a>
						<div class="dropdown-content">
							
							<a href="https://idec-teams.github.io/2023_NKLMI-NK_drug/documentation/notebook.html" class="dropdown-page cool-link2">
								Notebook
							</a>
						
							<a href="https://idec-teams.github.io/2023_NKLMI-NK_drug/documentation/protocol.html" class="dropdown-page cool-link2">
								Protocol
							</a>
						</div>
					</div>
				</div>
			</nav>
			<!------------ 页面标题PAGE TITLE -------------------->
			<div class="page-title">
				<div class="page-title-content">
					<!--<h1>PROTOCOL</h1>-->
					<!---Page title -->
				</div>
			</div>
		</header>
		<div class="grid-normal">
			<!--------------Introduction/Summary--------------------->
			<div class="introduction grid-normal">
			</div>
		</div>
		<div style="display:none" id="marker" class="triangle-down"> </div>
		<!------------------ BUTTONS ---------------------------->
		<div class="buttons">
			<div class="button-name">
			</div>
		</div>
		<div class="sections-container">
			<div class="carousel" style="margin:10%;">
				<h2>Here are our protocols: </h2>
				<p></p>
				<h2>This section shows details of the experiments that are not shown in detail in the notebook.</h2>
				<h2>(Click the block to view the hidden content) </h2>
				<p></p>
				<button type="button" class="collapsible">Mice and cell lines </button>
				<div class="content">

					<p style="font-size: 16px">Female NOD/ShiltJGpt-Prkdcem26Cd52Il2rg em26Cd22 /Gpt (NCG) mice were kept in specific pathogen-free conditions.<br>The tumor cells were grown in DMEM (Gibco), 10% fetal bovine serum (FBS). All cell lines tested negative for Mycoplasma.
</p>
				</div>
				<button type="button" class="collapsible">Lentivirus production</button>
				<div class="content" style="width:85%;margin:0 auto;height:207px;padding-top:20px">
					<p>Lentivirus production was achieved by utilizing lenti-X-293 cells. The day prior to transfection, lenti-X-293 cells were seeded in 15 cm-dish at the confluency of 70-80%. The media was substituted with 10.5 mL pre-warmed DMEM medium (Invitrogen) 2 hours before transfection. To each plate, the medium was mixed with pLVX-EF1a-IRESPuro or vector control plasmid, pCMVR8.74, pMD2.G and polybrene. Following a brief vortex, the mixture was incubated for 15 minutes at room temperature and then added drop by drop slowly to cells. 6 hours later, DMEM media was substituted with 20 mL pre-warmed DMEM media. Viral supernatant was gathered at 48 hours and 72 hours after transfection, subsequently filtered with a 0.40 mm filter (Fisher / VWR) to eliminate cell debris, and finally concentrated using AmiconUltra 100 kD ultracentrifugation units (Millipore). The virus was divided into smaller portions and stored at -80 ℃.</p>
				</div>
				<button type="button" class="collapsible">Library Virus Infection Experiment </button>
				<div class="content" style="width:85%;margin:0 auto;height:207px;padding-top:20px">
					<p>Infect 5×10 7~1×10 8 cells (245 mm × 245 mm plate to spread cells) with lentivirus at the functional MOI=0.4. Meanwhile, set up 2 parallel infection experiments as well as a blank control group (virus-free). 5 samples should be collected during the experiment, i.e. data on day 0 of control group, day 7 of control group and experimental group, day 14 of control group. Add the appropriate concentration of hygromycin B to each plate. When the control group is completely killed, digest the cells and collect at least 3×10 7 cells (this is the control sample on day 0, and about 200 μg of the genome can be extracted). The genomic DNA was extracted with the Tigen Genome Extraction Kit and the remaining cells were divided into two groups in equal amounts. Add the Vehicle and target drug to screen (each has 3 replicate wells) 2 days after passaging (after wall attachment). Collect the genome of the cells on day 7 and day 14 after drug treatment. Extract the cells genome and perform quality control.</p>
				</div>
				<button type="button" class="collapsible">Construction of AsPC-1 subcutaneous tumor model</button>
				<div class="content" style="width:85%;margin:0 auto;height:207px;padding-top:20px">
					<p>5×10 6 AsPC-1 cells in 50 μL of HBSS were injected via subcutaneous injection into the right flank of NCG mice. NK cells were treated with mitomycin-C to inhibit their proliferation Then intratumorally injected into subcutaneous AsPC-1 xenografts. After 72 hours, the persistence of NK cells in tumor tissues was collected and determined by immunofluorescence (IF). </p>
				</div>
				<button type="button" class="collapsible">Establishment of Orthotopic Pancreatic Cancer Murine Models</button>
				<div class="content" style="width:85%;margin:0 auto;height:267px;padding-top:20px">

					<p>The tumor graft was obtained by inoculating AsPC-1-luc cells subcutaneously into the right hind limbs of NCG mice (2 × 10 7 cells in each mouse) at 6 weeks of age . As the tumor reached a diameter of about 8 to 10 mm after tumor seeding, the mice were euthanized by cervical dislocation and tumor tissue was taken from the mice. After obtaining tumor samples, UW (University of Wisconsin) preservation solution was immediately applied. Following that, each tumor was wrapped in 5% low-melt agar and quickly sectioned with a tissue slicer at a thickness of 500 μm. As soon as the sections were obtained, they were immediately transferred to ice-cold UW 
solution. Using ophthalmic scissors, a 1 cm port was cut above the spleen, followed by the gentle squeeze out of the pancreas tail using atraumatic forceps. A tumor slice was carefully inserted into the subcapsular space after blunt dissection of the capsule of the pancreas. Vetbond (3M, 1469SB) was used to attach the graft to the pancreas,followed by 5-0 absorbable sutures for reinforcement. In the end, the abdominal cavity was closed after the spleen and pancreas were incorporated into it.

					</p>
				</div>
				<button type="button" class="collapsible">Tumor Tissue Dissociation</button>
				<div class="content" style="width:85%;margin:0 auto;height:207px;padding-top:20px">

					<p>On day 21 after tumor inoculation, the mice were euthanized by cervicaldislocation and tumor tissue was harvested. Cut tissue sample and digested by digestive solution (2.5 mg/ml type II collagenase and 10 μM Y-27632). Incubated tumor sample at 37℃ in the shaking incubator for thorough digestion about 1 hour until no obvious large tissue could be found. Then the digestion was filtrated by a 70 μm filter. The obtained filtrate was centrifuged at 1500 rpm for 5 min. Discarded the supernatant and added 3-5 mL of terminal culture media, centrifuged at 1500 rpm for 5 min. At last, used tumor culture media to resuspend cells. 
					</p>
				</div>
				<button type="button" class="collapsible">Sort NK cells</button>
				<div class="content" style="width:85%;margin:0 auto;height:207px;padding-top:20px">

					<p>Cells were collected and rinsed once using MACS buffer (0.5 % BSA and 2 mM EDTA in PBS). The NK cells were subsequently sorted by MojoSort™ Human NK Cell Isolation Kit (Biolegend, 480054). 
					</p>
				</div>
				<button type="button" class="collapsible">CRISPR screening </button>
				<div class="content" style="width:85%;margin:0 auto;height:207px;padding-top:20px">

					<p>Total RNA isolation, RNA quality control, library construction, sequencing and data analysis were performed by Shanghai Biotechnology Corporation (Shanghai, China). Phusion Flash High Fidelity Master Mix (Thermo Fisher) to perform two-step PCR amplification for dgRNA readout. PCR #1 used primers to amplify dgRNA Cassette: <br/>
Forward: 5’-AATGGACTATCATATGCTTACCGTAACTTGAAAGTATTTCG-3’ <br/> 
Reverse: <br/>
5’AACGTTCACGGCGACTACTGCACTTATATACGGTTCTC-3’  <br/>
PCR #2 used uniquely barcoded primers:  <br/>
Forward:  <br/>
5’-TCTTGTGGAAAGGACGAAACACC-3’  <br/>
Reverse:  <br/>
5’-GCCAAGTTGATAACGGACTAGCCTT-3’ 

					</p>
				</div>
				<button type="button" class="collapsible">CAR-NK-92 Cell Line Establishment  </button>
				<div class="content" style="width:85%;margin:0 auto;height:337px;padding-top:20px">

					<p>Seed NK-92 cells at 1×105 per well in a 24-well plate. Mix 2 mg/mL of 
polybrene per well, and add 2 mg/mL of polybrene per well, mix thoroughly, and add the encapsulated virus at different MOI values (1, 10, 50, 100, 500, 1000). After 12 hours of infection, all the cells were aspirated and transferred to 25 cm2 containing 9 mL of complete culture medium. The percentage of CAR-positive cells was detected by FACS. The cells were collected by centrifugation at 3000 rpm for 5 min. Discard the supernatant and add ice-cold PBS 1 mL to wash the cells once. Cells were resuspended in 50 μL PBS and 2 μL of FLAG-APC antibody (BioLegend, 637308)was added. Incubate NK cells at room temperature for 15 min, add 1 mL of ice-cold 
PBS and centrifuge at 3000 rpm for 5 min to collect the cells. Add 1 mL ice-cold PBS and wash the cells once, discard the supernatant and add 50 μL ice-cold PBS to resuspend the cells for flow assay. Cells with the highest CAR positivity rate and cell viability of not less than 95% are selected for expansion and sorting. The cells were sorted after proliferation to 10 7 and the cells were centrifuged at 1500 rpm for 5 min, and washed once with 30 mL of ice-cold PBS. Cells were resuspended in 200 μL PBS, labeled with 10 μL of sterile FLAG-APC antibody (BioLegend, 637308), then incubated for 15 min away from light. The cells were washed with 30 mL of ice-cold PBS and the supernatant was well discarded. 500 μL PBS was used to resuspend the 
cells. Positive cells were sorted. Sorted cells were cultured in 24-well plates, 
expanded and get subclone. 
</p>
				</div>
							<button type="button" class="collapsible">IF</button>
				<div class="content" style="width:85%;margin:0 auto;height:207px;padding-top:20px">

					<p>Tumor tissues were formalin-fixed and paraffin-embedded. Incubated with fluorescence-conjugated antibody targeting CD56 (NCAM1 (CD56) (E7X9M) Rabbit mAb, CST, 99746). Then, incubated with fluorescence conjugated antibodies. Images were captured with a 63×/1.4-NA oil-immersion objective. Leica confocal software (Leica, Germany) was used to process images. 
</p>
				</div>
							<button type="button" class="collapsible">ELISA</button>
				<div class="content" style="width:85%;margin:0 auto;height:207px;padding-top:20px">

					<p>4 hours co-culture of NK cells with tumor cells, the supernatant was collected and analyzed using the quantikine ELISA human IFN-γ Immunoassay according to the manufacturer’s instructions. A microplate reader (PerkinElmer, Waltham, MA, USA) was used to read the 96-well plates at 450 nm.
</p>
				</div>
							<button type="button" class="collapsible">Cytotoxicity assay</button>
				<div class="content" style="width:85%;margin:0 auto;height:207px;padding-top:20px">

					<p>For lactate dehydrogenase (LDH), CytoTox96 cytotoxicity assay was used 
according to the manufacturer's instructions. Briefly, target cells were plated in NK cells media in white-walled 96-well plates, followed by the addition of NK cells at E:T ratio=1:1 or 5:1. Finally, cytotoxicity was calculated based on LDH release using the following formula: Cytotoxicity (%) = [LDHE:T -LDHE ]/LDHMax× 100%. Specific steps as we previously described. 

</p>
				</div>
							<button type="button" class="collapsible">ATP generation assay</button>
				<div class="content" style="width:85%;margin:0 auto;height:207px;padding-top:20px">

					<p>ATP generation analyzed using the Luminescent Cell Viability Assay according to the manufacturer’s instructions. In brief, thaw the CellTiter-Glo reagent and equilibrate to RT. Add the test compound to experimental wells, then add equivalent volume CellTiter-Glo Reagent to the cell culture medium present in each well. Agitate the mixture for 2 minutes on an orbital shaker to initiate the lysis of cells. Total luminescence signal was quantified using Living Image software (Perkin Elmer). 


</p>
				</div>
							<button type="button" class="collapsible">Seahorse metabolic assays</button>
				<div class="content" style="width:85%;margin:0 auto;height:207px;padding-top:20px">

					<p>Specific steps as the instruction described and analyzed on an Agilent Seahorse XF 24-well analyzer. Briefly, after co-cultured with tumor cells, NK-92, BBζ cells and BBζ-M NK cells were seeded at 2 × 10 5 per well in cell culture microplates pre-coating with poly-lysine. Cells were equilibrated for 1 hour in XF assay medium supplemented with 10 mM glucose, 1 mM sodium pyruvate and 2 mM glutamine in a non-CO2 incubator. OCR were monitored at baseline and throughout sequential injections of oligomycin (1.5 μM), carbonyl cyanide-4-(trifluoromethoxy) phenylhydrazone (0.5 μM) and rotenone or antimycin A (0.5 μM each). 
</p>
				</div>
					<button type="button" class="collapsible">Statistical analysis</button>
				<div class="content" style="width:85%;margin:0 auto;height:207px;padding-top:20px">

					<p>GraphPad Prism 8.0 was used for all statistical analyses. A two-tailed unpaired student t-test was used to determine significance. One-way analysis of variance (ANOVA) with a Bonferroni post-test was used to compare differences among multiple groups. Survival analysis was performed by KaplanMeier survival analysis.
</p>
				</div>
				<script>
					var coll = document.getElementsByClassName("collapsible");
					var i;

					for (i = 0; i < coll.length; i++) {
						coll[i].addEventListener("click", function() {
							this.classList.toggle("active");
							var content = this.nextElementSibling;
							if (content.style.display === "block") {
								content.style.display = "none";
							} else {
								content.style.display = "block";
							}
						});
					}
				</script>
			</div>
		</div>
		</section>
		</div>
		<a href="#" id="mobile-to-top-arrow" class="to-top">
		
			<img style="max-height: 90px" src="../images/top.png">
		</a>
		<!---------- 页脚FOOTER ------------>
		    <footer>
        <!-- 头部波浪效果 -->
        <div class="footer-header" style="background:white">
            <svg class="footer-waves" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"
                viewBox="0 24 150 28" preserveAspectRatio="none" shape-rendering="auto">
                <defs>
                    <path id="footer-gentle-wave"
                        d="M-160 44c30 0 58-18 88-18s 58 18 88 18 58-18 88-18 58 18 88 18 v44h-352z" />
                </defs>
                <g class="footer-parallax">
                    <use xlink:href="#footer-gentle-wave" x="48" y="0" fill="#8FC4E3" />
                    <use xlink:href="#footer-gentle-wave" x="48" y="3" fill="#FFBD80" />
                    <use xlink:href="#footer-gentle-wave" x="48" y="5" fill="#FFED60" />
                    <use xlink:href="#footer-gentle-wave" x="48" y="7" fill="#F6D1D9" />
                </g>
            </svg>
        </div>
    </footer>
		<script src="../js/jQuery.js" type=text/javascript></script>
		<script src="../js/detect.js" type="text/javascript"></script>
		<script src="../js/main_javascript.js" type=text/javascript></script>
		<script src="../js/MoveContent.js" type="text/javascript"></script>

		<script>
			document.addEventListener("DOMContentLoaded", function(event) {
				MoveContentOutside();
			});
		</script>
			 	<script src="../js/loader.js" type="text/javascript"></script>
	</body>
</html>
>>